### PR TITLE
Add policyViolationContextLimit on GRC Ansible automation UI

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo <chuluo@redhat.com>

https://github.com/stolostron/backlog/issues/26436

Depends on https://github.com/stolostron/multiclusterhub-operator/pull/886

Screen Shot 2022-10-19 at 1 02 19 PM

### CODE REVIEW

**Comment by ChunxiAlexLuo** [comment]
Screen Shot 2022-10-19 at 1 02 19 PM

File: frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx [hunk 1]
```diff
@@ -195,6 +195,13 @@ export function PolicyAutomationWizard(props: {
                         placeholder="Add variable"
                         hidden={(item) => !item.spec?.automationDef?.name}
                     />
+                    <WizNumberInput
+                        path="spec.automationDef.policyViolationContextLimit"
+                        label="Policy Violation Context Limit"
+                        labelHelp="The maximum number of the non-compliant replicated policies 
+                        that will be passed to the Ansible platform in the extra variables for 
+                        a root policy. When it is set to 0, it means no limit and default value is 1000."
```
Comment (mprahl) [inline frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx:203]:
```suggestion
                        labelHelp="The maximum number of non-compliant cluster policy details
                        that will be passed to the Ansible platform as extra variables for 
                        the policy. When it is set to 0, it means no limit and default value is 1000."
```

File: frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx [hunk 2]
```diff
@@ -195,6 +195,13 @@ export function PolicyAutomationWizard(props: {
                         placeholder="Add variable"
                         hidden={(item) => !item.spec?.automationDef?.name}
                     />
+                    <WizNumberInput
+                        path="spec.automationDef.policyViolationContextLimit"
+                        label="Policy Violation Context Limit"
+                        labelHelp="The maximum number of non-compliant cluster policy details
+                        that will be passed to the Ansible platform as extra variables for 
+                        the policy. When it is set to 0, it means no limit and default value is 1000."
```
Comment (mprahl) [inline frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx:203]:
```suggestion
                        that will be passed to the Ansible platform as extra variables. When it is
                        set to 0, it means no limit. The default value is 1000."
```

**Comment by mprahl** [comment]
@dockerymick could you please review this help text?

**Comment by ChunxiAlexLuo** [comment]
> Thanks for pulling me in ! @ChunxiAlexLuo I left a suggestion

Done^^

**Review by mprahl — APPROVED** [review]

**Comment by KevinFCormier** [comment]
FYI @RobDolares You'll need to externalize this one when you perform your merge/rebase.

**Pull request merged**